### ISSUE TITLE
Paypal troubles

### ISSUE THREAD

**Issue by brokkr** [liberapay/liberapay.com#2142]
I am at a loss to explain my experience but I will try. I know I should try to extract each issue on it's own and file separate issues but I wouldn't know where to start. So I'm just going to share for what it's worth.

I recently convinced an otherwise stalwart Patreon/Paypal only podcaster to give liberapay a go. They set up an account but something was different compared to most of the accounts I donate to because apparently it had to go through paypal (there is a small PP logo next to it).

First: I pick the level I want to donate at and expect to be asked for the frequency of payments (weekly, monthly, yearly). This question is never asked and it seems like I'm auto-stuck with weekly which is a terrible option. I try to cancel this donation.

Next: I set up a custom donation, specifying how much I want to donate on an explicitly yearly basis. Despite me giving the go-ahead, the giving page says "awaiting payment" which is baffling to say the least. I understand now that PayPal wants me to interact directly with them - as little as I want to - but "awaiting payment" did not communicate this to me at all. I expected - given my experience with liberapay - that it was a technicality and I just needed to be patient. Logging in a week later, I see it's still "awaiting payment". At the very least, this is a UI problem. 

Note: If Paypal cannot autorenew, why am I still given the option to do pay manually or autorenew?

Then: I click through to Paypal. This has so far presented two problems. One is that the Paypal site seems to suggest that I can pay using a credit card, i.e. no Paypal account. This is a lie. I am not allowed to "accept and pay" without filling in the "create an account" section. I searched that screen from top to bottom for an elusive "pay without making an account" button but I didn't find it. 

The other, and even more aggravating, is that despite completing every part of paypal's requirements as well as VISA's, and the payment supposedly completing, Liberapay still says

    secret donation to latenightlinux (1 year of €40.00) — failed — PayPal status code: aborted by payer 	€40.00 		
    charge — failed — error message: aborted by payer

I need to be absolutely clear: This is not true. I did not abort. I was asked for three confirmation codes. I filled in three confirmation codes. To the best of my knowledge, this transaction was completed. Afterwards, I received an email from paypal asking me to confirm my email which I have now done. I can also see that they have added my card to the account, something that I did not ask them to, but that is neither here nor there.

Finally: In Giving / schedule I have a payment due on April 19th (maybe a month after first attempt?) for 11.44£. I have no idea why this is there. I never picked donations in pounds and 11.44£ doesn't correspond to either any monthly or yearly amount I have tried donating. It certainly does not correspond to any schedule currently set up (40€ / year).

Finally 2: Liberapay insists that I have _two_ donations "awaiting payment". Clicking on the notice takes me to the one, single paypal payment button for this donation. I have a feeling that it's the original, bad, weekly donation to the same creator that is somehow unkillable and haunting me but I since I cannot see this second donation, I have no way of knowing for sure.

In conclusion: This feels like a disaster. I'm sure it's Paypal's fault not yours. But something is well and truly borked with everything about this linkage and if it works this badly I wish you would not offer paypal as an option to recipients.

**Comment by Changaco** [comment]
> I recently convinced an otherwise stalwart Patreon/Paypal only podcaster to give liberapay a go. They set up an account but something was different compared to most of the accounts I donate to because apparently it had to go through paypal (there is a small PP logo next to it).

The payment had to go through PayPal because the recipient didn't connect a Stripe account, despite being in a country where Stripe is available.

> I need to be absolutely clear: This is not true. I did not abort.

You definitely aborted that specific payment attempt, it doesn't even exist in PayPal's database anymore. However, since you say that you completed a payment, I'm guessing that you did it by going directly to PayPal's website, and in that case Liberapay can't see the payment at all.

> In Giving / schedule I have a payment due on April 19th (maybe a month after first attempt?) for 11.44£. I have no idea why this is there.

This scheduled payment is for a different recipient, it's the renewal of a donation that you started last year.

**Comment by Changaco** [comment]
I have made two changes to address this issue:

- The imprecise label “Awaiting payment” in the Giving page has been replaced with “You need to initiate a new payment to fund this donation. Click on the button below to proceed.”
- The misleading sentence “For now this donation can only be renewed externally through PayPal.” in the payment page has been replaced with “PayPal is currently the only available option to fund this donation. Please click on the following button to proceed:”

**Comment by Benianus3D** [comment]
Why PayPal made accounts permanently limited because of Liberapay transactions, I've this problem two-time consecutively
and I'm obliged to open another account each time
I hope this problem will be resolved
